src/libs/ReportUtils.js

```diff
@@ -1089,6 +1100,28 @@ function getChatRoomSubtitle(report) {
     return getPolicyName(report);
 }
 
+/**
+ * Get the subtitle link for the report
+ * @param {Object} report
+ * @returns {String}
+ */
+function getChatRoomSubtitleLink(report) {
```

Comment:
We already have a `getChatRoomSubtitle` see [here](https://github.com/Expensify/App/blob/f10878fd12b5b47861a91d8684e9d76eebe0fee7/src/components/AvatarWithDisplayName.js#L57). This once is a bit different since it gives us the text for parent navigation and we use it few lines below [here](https://github.com/Expensify/App/blob/f10878fd12b5b47861a91d8684e9d76eebe0fee7/src/components/AvatarWithDisplayName.js#L63).

I agree it returns just a string and not a link. I'll rename this to `getParentNavigationText`

Reply:
updated